include/seqan3/range/decorator/gap_decorator_anchor_set.hpp

```diff
@@ -98,8 +103,8 @@ class gap_decorator_anchor_set
         //!\brief A pointer to the current anchor gap node. Note that the current tuple value at position 0 is the
         //!       start of the right gap that is still behind the current iterator position.
         typename gap_decorator_anchor_set::set_iterator_type anchor_set_it{};
-        //!\brief Caches the current value pointed at.
-        typename gap_decorator_anchor_set::value_type current_val{gap{}};
```

Comment:
Hm, is the iterator *on* a gap or *at* a gap? :thinking:

Reply:
I'd say `at`... but I'm not sure

Reply:
`points_to_gap`?

Reply:
Why not `is_gap`. Is it public API?

Reply:
Is this public API? If not who cares?